_data/sidebars/home_sidebar.yml

```diff
@@ -154,25 +154,25 @@
         url: /cloud/aws/ecs.html
       - title: Amazon Elastic Kubernetes Service (EKS)
         url: /cloud/aws/eks.html
-      - title: KOPS
+      - title: Kubernetes Operations (KOPS)
```

Comment:
change to `KOPS (Kubernetes Operations)` since kops is the more well known name

Reply:
Updated as per comment.